Makefile

```diff
@@ -10,7 +10,7 @@ clean:
 	@rm -rf controller/controller
 
 build:
-	@cd controller && godep go build -a -tags "netgo static_build" -installsuffix netgo -ldflags "-w -X github.com/shipyard/shipyard/version.GitCommit=$(COMMIT)" .
+	@cd controller && godep go build -a -tags "netgo static_build" -installsuffix netgo -ldflags "-w -X github.com/shipyard/shipyard/version.GitCommit $(COMMIT)" .
```

Comment:
You shouldn't get an error and yes you should be using Go 1.5. Thanks!
On Dec 17, 2015 12:38 AM, "yangdongsheng" <notifications@github.com> wrote:

> In Makefile
> <https://github.com/shipyard/shipyard/pull/705#discussion_r47871188>:
>
> > @@ -10,7 +10,7 @@ clean:
> >  	@rm -rf controller/controller
> >
> >  build:
> > -	@cd controller && godep go build -a -tags "netgo static_build" -installsuffix netgo -ldflags "-w -X github.com/shipyard/shipyard/version.GitCommit=$(COMMIT)" .
> > +	@cd controller && godep go build -a -tags "netgo static_build" -installsuffix netgo -ldflags "-w -X github.com/shipyard/shipyard/version.GitCommit $(COMMIT)" .
>
> But, we got an error in "make image" with =:
> go version
>
> go version go1.4.2 linux/amd64
>
> Do you mean I need to upgrade my golang?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shipyard/shipyard/pull/705/files#r47871188>.
>